composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "config": {
         "platform": {
-            "php": "7.0.22"
+            "php": "7.2"
```

Comment:
I feel like in cases like this, where it would be possible for somebody to be loading the theme as a composer package, we should specify a PHP version in `require` as well as a platform. If someone isn't running a high enough version of PHP, the failure should happen when they run `composer install` rather than when they try to actually run the theme code.